datumaro/plugins/open_images_format.py

```diff
@@ -50,9 +54,26 @@ class UnsupportedSubsetNameError(DatasetError):
     def __str__(self):
         return "Item %s has an unsupported subset name %r." % (self.item_id, self.subset)
 
+@attrs(auto_attribs=True)
+class UnsupportedBoxIdError(DatasetError):
+    box_id: str
+
+    def __str__(self):
+        return "Item %s has a mask with an unsupported box ID %r." % (
+            self.item_id, self.box_id)
+
+@attrs(auto_attribs=True)
+class UnsupportedMaskPathError(DatasetError):
```

Comment:
Consider moving these errors to `errors.py`.

Reply:
> Consider moving these errors to components/errors.py.

I think that would be done best in the scope of #313.

> BTW, as of 21.1.0 of attrs, auto_attribs can be replaced with define. and it looks like a thing I'd want to start using in the project.

I agree, but _that_ would be done best in the scope of #310. 🙂